packages/interop/src/json.spec.ts

```diff
@@ -21,7 +21,7 @@ describe('@helia/verified-fetch - json', () => {
       await loadFixtureDataCar(controller, 'QmQJ8fxavY54CUsxMSx9aE9Rdcmvhx8awJK2jzJp4iAqCr-tokens.uniswap.org-2024-01-18.car')
       verifiedFetch = await createVerifiedFetch({
         gateways: [`http://${controller.api.gatewayHost}:${controller.api.gatewayPort}`],
-        routers: [`http://${controller.api.gatewayHost}:${controller.api.gatewayPort}`]
+        routers: []
```

Comment:
temp fix to stop endless routing caused by https://github.com/ipfs/helia/issues/501